api/api_storage.go

```diff
@@ -41,6 +41,7 @@ import (
 // StorageMiner is a low-level interface to the Filecoin network storage miner node
 type StorageMiner interface {
 	Common
+	Net
```

Comment:
I don't think we have to do this in initial MRA split PR, but ideally:
* We'd have the miner interface split into more, per service interfaces
* In the API serve logic we'd only construct the RPC server with the services a node actually supports